src/main/java/duke/EmptyDoneException.java

```diff
@@ -0,0 +1,18 @@
+package duke;
+
+public class EmptyDoneException extends Exception {
+    /**
+     *
+     */
+    private static final long serialVersionUID = 1L;
+    private final static String errorMessage = "OOPS!!! The description of a done cannot be empty. ";
```

Comment:
I think the name of the constant should be in capital form. I noticed the same issue in several other places too.